docs/content/direct/core-chart-step-by-step-installation.md

```diff
@@ -0,0 +1,340 @@
+ ✓ Installing CNI 🔌 
+ ✓ Installing StorageClass 💾 
+Set kubectl context to "kind-kubeflex"
+You can now use your cluster with:
+
+kubectl cluster-info --context kind-kubeflex
+
+Have a question, bug, or feature request? Let us know! https://kind.sigs.k8s.io/#community 🙂
+Installing an nginx ingress...
+namespace/ingress-nginx created
+serviceaccount/ingress-nginx created
+serviceaccount/ingress-nginx-admission created
+role.rbac.authorization.k8s.io/ingress-nginx created
+role.rbac.authorization.k8s.io/ingress-nginx-admission created
+clusterrole.rbac.authorization.k8s.io/ingress-nginx created
+clusterrole.rbac.authorization.k8s.io/ingress-nginx-admission created
+rolebinding.rbac.authorization.k8s.io/ingress-nginx created
+rolebinding.rbac.authorization.k8s.io/ingress-nginx-admission created
+clusterrolebinding.rbac.authorization.k8s.io/ingress-nginx created
+clusterrolebinding.rbac.authorization.k8s.io/ingress-nginx-admission created
+configmap/ingress-nginx-controller created
+service/ingress-nginx-controller created
+service/ingress-nginx-controller-admission created
+deployment.apps/ingress-nginx-controller created
+job.batch/ingress-nginx-admission-create created
+job.batch/ingress-nginx-admission-patch created
+ingressclass.networking.k8s.io/nginx created
+validatingwebhookconfiguration.admissionregistration.k8s.io/ingress-nginx-admission created
+Patching nginx ingress to enable SSL passthrough...
+deployment.apps/ingress-nginx-controller patched
+Waiting for nginx ingress with SSL passthrough to be ready...
+pod/ingress-nginx-controller-68494d6d4-4qhmv condition met
+Setting context to "kind-kubeflex"...
+Switched to context "kind-kubeflex".
+```
+
+## Step 2: Check Container Name
+
+verify the container name that will be needed for the Helm chart configuration:
+
+```bash
+docker ps --filter "name=kubeflex" --format "{{.Names}}"
+```
+
+**Expected output (similar to):**
+```
+kubeflex-control-plane
+```
+
+## Step 3: Update Chart Dependencies
+
+navigate to your KubeStellar directory and update the chart dependencies:
+
+```bash
+helm dependency update core-chart
+```
+
+**Expected output (similar to):**
+```
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "antrea" chart repository
+...Successfully got an update from the "prometheus-community" chart repository
+...Successfully got an update from the "istio" chart repository
+Update Complete. ⎈Happy Helming!⎈
+Saving 2 charts
+Downloading kubeflex-operator from repo oci://ghcr.io/kubestellar/kubeflex/chart
+Pulled: ghcr.io/kubestellar/kubeflex/chart/kubeflex-operator:v0.8.9
+Digest: sha256:2be43de71425ad682edca6544f6c3a5864afbfad09a4b7e1e57bde6dae664334
+Downloading argo-cd from repo oci://ghcr.io/argoproj/argo-helm
+Pulled: ghcr.io/argoproj/argo-helm/argo-cd:7.8.5
+Digest: sha256:662f4687e8e525f86ff9305020632b337a09ffacb7b61b7c42a841922c91da7b
+Deleting outdated charts
+```
+
+## Step 4: Initial Chart Installation
```

Comment:
Why are steps 4 and 5 here? Why not proceed directly to the full `helm upgrade --install` command?